presto-pinot-toolkit/pom.xml

```diff
@@ -641,6 +645,12 @@
             <groupId>commons-codec</groupId>
             <artifactId>commons-codec</artifactId>
         </dependency>
+
+        <dependency>
+            <groupId>org.apache.lucene</groupId>
+            <artifactId>lucene-core</artifactId>
+            <version>8.10.0</version>
```

Comment:
you shouldn't need the `<version>` tag in here if the `luecene-core` dependency is defined in the root pom's `dependencyManagement` section. I would recommend adding it to the root pom's `dependencyManagement` section with the `dep.lucene.version` variable there and remove the `<version>` tag here.